site/content/integrate/marketplace-submissions/_index.md

```diff
@@ -0,0 +1,60 @@
+weight: 100
+aliases: /extend/plugins/community-plugin-marketplace/
+---
+
+The [Mattermost web Marketplace](https://mattermost.com/marketplace/) supports discovery, installation, and updates of extensions of the Mattermost platform. If you have built a plugin, app, playbook, integration, or tool that helps other users get more from Mattermost, the web Marketplace is a great way to get feedback on your contribution and help make it more popular. Once your submission is accepted to the Web Marketplace, Mattermost will also send you swag!
+
+All contributions are eligible to be added to our web marketplace at mattermost.com/marketplace after a review from our team. Apps and plugins that have been thoroughly reviewed by Mattermost will be eligible for our in-product marketplace as well with some additional requirements. Our in-product Marketplace enables customers to install and manage their apps and plugins from directly within the product. 
+
+## Type of integrations to submit to the web marketplace
+
+- Apps
+- Plugins
+- Webhook configuration
+- Playbook templates
+- Utility tools, such as importers, command line interfaces, and scripts
+
+## How to submit your contribution for consideration for the web marketplace
+
+Please fill out the information in [this form](https://forms.gle/PE8kmfSuneP9GWnq8) to be reviewed by a member of the Mattermost team. 
+
+Every contribution will go through the following checklist before being added to the web marketplace at mattermost.com/marketplace:  
+
+1. There is a link to the contribution so we can test it out.
+2. There is documentation to support people installing, configuring, and using the contribution. We also recommend including screenshots to give users a better understanding of the workflow and user experience. 
+3. There is a public issue or bug tracker for users to report bugs or issues they encounter. 
+
+We will also reach out to you to learn more about the integration, your experience developing with Mattermost, and to coordinate postings for social media. 
+
+## How to submit your contribution for consideration for the in-product marketplace
+
+There are additional functional, technical, and security requirements for apps and plugins to be considered for the in-product marketplace. You can check off these items when you submit your contributions by opening an [issue in our in-product marketplace repository](https://github.com/mattermost/mattermost-marketplace/issues/new?assignees=hanzei&labels=Plugin%2FNew&template=add_plugin.md&title=Add+%24REPOSITORY_NAME+to+Marketplace). 
+
+### Technical requirements
+
+1. The plugin or app works for 60,000 concurrent connections and within a High Availability deployment. **Note:** There are currently no publicly-available tools available to verify these properties. As such, they are checked during code review by a developer.
```

Comment:
Is there any guidance on how a "High Availability deployment" is defined?

Reply:
I've removed this section per your suggestion below. :slightly_smiling_face: